src/main/java/net/dv8tion/jda/api/entities/EventType.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.entities;
+
+/**
+ * Represents the type of event which will cause the auto moderation system to check for a specific trigger.
+ */
+public enum EventType {
```

Comment:
All automod entity types should be moved to a dedicated package under entities. `net.dv8tion.jda.api.entities.automod`